### ISSUE TITLE
Materials / cubemap / dynamic is unnecessarily complex

### ISSUE THREAD

**Issue by looeee** [mrdoob/three.js#13223]
I want to refactor this [example](https://threejs.org/examples/?q=dyna#webgl_materials_cubemap_dynamic) to remove the use of `BinaryLoader`, but it seems to me that it is far more complex than is neccessary to demonstrate a dynamic cube map. 

I'd suggest two option:

1. remove the example and replace it with `materials / cubemap / dynamic2` which is much simpler. 

2. simplify it to use a single car model with built in materials, a single camera that tracks the car, no day/night switch, no lens flare, no motion blur. 

There is also a [Car.js](https://github.com/mrdoob/three.js/blob/dev/examples/js/Car.js) file used only in this example, which can be used to load and create a car in either JSON or JSON Binary format. We should either remove this or refactor it so that it can be used with any model format.

**Comment by Mugen87** [comment]
I think i tend to option two. The handling of the car is actually quite funny 😁

**Comment by looeee** [comment]
OK, in that case we will need to convert one of the current car models in JSON Binary format into two models in another format (front wheels and rest of car). 

I guess it should be GLTF?

**Comment by mrdoob** [comment]
> 1. remove the example and replace it with `materials / cubemap / dynamic2` which is much simpler. 

👍

However, I think we should save some of the car handling code though. Should we create a new example `webgl_interactive_car` maybe?

> OK, in that case we will need to convert one of the current car models in JSON Binary format into two models in another format (front wheels and rest of car).

Yeah, but I would leave that for another PR.

**Comment by looeee** [comment]
How about `misc_controls_car`?

>Yeah, but I would leave that for another PR. 

OK, however the only thing preventing removal of `BinaryLoader` at this point is the car models. 
There are 4 cars, which seems excessive - I think we should remove all but one. 

I've already converted the Ferrari model to GLTF in #13084. How about I set it up so that it can be used in the `interactive / car controls` example as well?
